test/JACCTests.jl

```diff
@@ -3,18 +3,16 @@ module JACCTests
 import JACC
 using ReTest
 
-const backend = JACC.JACCPreferences.backend
-
-@static if backend == "cuda"
+@static if JACC.backend == "cuda"
```

Comment:
Should we replace this with the preferred macro `JACC.@init_backend`

Reply:
We can remove the `using ...` statements, but this code is including the backend-specific test files.

Reply:
OK, I see thanks! Your call.